.distignore

```diff
@@ -1,22 +1,13 @@
 # A set of files you probably don't want in your WordPress.org distribution
-.distignore
-.editorconfig
-.git
-.gitignore
-.travis.yml
-.DS_Store
-Thumbs.db
-bin
-behat.yml
-circle.yml
+.*
+*.xml
+*.yml
 Gruntfile.js
 package.json
-phpunit.xml
-multisite.xml
 phpunit.xml.dist
-phpcs.ruleset.xml
-README.md
-wp-cli.local.yml
-tests
-/vendor/lcache/lcache/tests
-node_modules
```

Comment:
@danielbachhuber I actually didn't, GitHub is just making it look like I did 😸 

https://github.com/fjarrett/wp-lcache/blob/2a58d664dc0a4211ea159599f09608adf5237c60/.distignore